packages/extension-commerce-bm-seo/package.json

```
@@ -1,40 +1,37 @@
 {
   "name": "@salesforce/extension-commerce-bm-seo",
-  "version": "0.1.0-extensibility-preview.4",
+  "version": "0.1.0-extensibility-preview.5",
   "private": false,
   "scripts": {
     "lint": "npm run lint:js",
     "lint:fix": "npm run lint:js -- --fix",
     "lint:js": "pwa-kit-dev lint \"**/*.{js,jsx,ts,tsx}\"",
     "start": "npm --prefix ./dev start",
-    "start:inspect": "npm --prefix ./dev run start:inspect"
+    "start:inspect": "npm --prefix ./dev run start:inspect",
+    "test": "pwa-kit-dev test"
   },
-  "peerDependencies": {
+  "devDependencies": {
     "@loadable/component": "^5.15.3",
-    "@salesforce/pwa-kit-extension-sdk": "4.0.0-extensibility-preview.4",
-    "@salesforce/pwa-kit-react-sdk": "4.0.0-extensibility-preview.4",
+    "@salesforce/pwa-kit-dev": "4.0.0-extensibility-preview.5",
+    "@salesforce/pwa-kit-extension-sdk": "4.0.0-extensibility-preview.5",
+    "@salesforce/pwa-kit-react-sdk": "4.0.0-extensibility-preview.5",
+    "@salesforce/pwa-kit-runtime": "4.0.0-extensibility-preview.5",
     "@salesforce/commerce-sdk-react": "^3.2.0-extensibility-preview.4",
```

I cannot build/run this PR via `npm ci` at the root. Seeing this error, which is also similar to what's seen in CI:
```
erna ERR! lerna ENOENT: no such file or directory, open '/Users/vmarta/repos/pwa-kit/packages/extension-commerce-bm-seo/node_modules/.bin/pwa-kit-extension-sdk'
node:child_process:957
    throw err;
    ^
```

Could it be because of the changes in this package.json file?

For example, try updating this version number, since `preview.5` is the latest version for it. 
```suggestion
    "@salesforce/commerce-sdk-react": "^3.2.0-extensibility-preview.5",
```

It was the older `preview.4` because it used to be under the `peerDependencies`. @kevinxh was saying that it's a known issue with our monorepo... the peer dep should be a version that's already published to npm.